api/src/main/java/org/openmrs/module/dhisconnector/api/db/DHISConnectorDAO.java

```diff
@@ -39,4 +40,8 @@
 	void saveLocationToOrgUnitMapping(LocationToOrgUnitMapping locationToOrgUnitMapping);
 
 	void deleteLocationToOrgUnitMappingsByLocation(Location location);
+
+	void saveSerializedObject(SerializedObject serializedObject);
```

Comment:
thanks @Gravewalker666  could you add tests for these methods

Reply:
@HerbertYiga for dao methods?

Reply:
yes, basing on their implementation

Reply:
Sure @HerbertYiga 🙂

Reply:
@HerbertYiga I included a test for dhis mapping sharing and updated the PR.
Can you please review it again?